x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/summary_actions/check_all/check_index.ts

```diff
@@ -41,7 +42,7 @@ export async function checkIndex({
   abortController: AbortController;
   batchId: string;
   checkAllStartTime: number;
-  ecsMetadata: Record<string, EcsMetadata> | null;
```

Comment:
For reviewers:

`ecsMetadata` represents `EcsFlat` object which is imported statically so it can never be `null`.